samples/apps/copilot-chat-app/webapp/src/components/chat/chat-history/ChatHistoryItem.tsx

```diff
@@ -91,6 +101,18 @@ export const ChatHistoryItem: React.FC<ChatHistoryItemProps> = ({ message, getRe
         content = <PlanViewer message={message} messageIndex={messageIndex} getResponse={getResponse} />;
     } else if (message.type === ChatMessageType.Document) {
         content = <ChatHistoryDocumentContent isMe={isMe} message={message} />;
+    } else if (message.content.startsWith('data:image')) {
+        content = (
+            <img
+                className={
+                    features[FeatureKeys.AzureContentSafety].enabled && isModerating
+                        ? mergeClasses(classes.image, classes.blur)
+                        : classes.image
+                }
+                src={message.content}
+                alt="TBA"
+            />
```

Comment:
can we create a `ChatHistoryImageContent` component for this for consistency and encapsulation sake?

Reply:
This is no longer relevant - only treating uploaded images as docs